update.php

```diff
@@ -137,6 +137,19 @@
 			'download_url' => 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar',
 		),
 	),
+	'7.4' => array(
+		'php' => array(
+			'base_name'       => 'php:7.4-rc-fpm',
```

Comment:
```suggestion
			'base_name'       => 'php:7.4.0beta1-fpm',
```

See [existing comment](https://github.com/WordPress/wpdev-docker-images/pull/1/files#r310900656)